apps/server/src/modules/files-storage/uc/files-storage.uc.spec.ts

```diff
@@ -199,374 +170,533 @@ describe('FilesStorageUC', () => {
 		expect(filesStorageUC).toBeDefined();
 	});
 
-	describe('upload from link()', () => {
-		beforeEach(() => {
-			httpService.get.mockReturnValue(
-				createObservable(
-					{},
-					{
-						connection: 'keep-alive',
-						'content-length': '10699',
-						'content-type': 'image/jpeg',
-					}
-				)
-			);
-		});
+	describe('uploadFromUrl is called', () => {
+		const createResponse = (data: Readable, headers: AxiosResponseHeaders = {}): AxiosResponse<Readable> => {
```

Comment:
douplicated helper function?  Same directly return vs first adding to const.

Reply:
Removed duplicate and direct return